README.md

```diff
@@ -1,108 +1,108 @@
+     - [Running From The Command Line](#running-from-the-command-line)
+       - [Diagnostic Types](#diagnostic-types)
+       - [Standard Options](#standard-options)
+       - [PKI Authentication Options](#pki-authentication-options)
+       - [Http Proxy Options](#http-proxy-options)
+       - [Remote Execution Options](#remote-execution-options)
+       - [Usage Examples](#usage-examples)
+       - [Customizing What Is Collected](#customizing-what-is-collected)
+           - [The Config Directory](#the-config-directory)
+           - [Removing Or Modifying Calls](#removing-or-modifying-calls)
+           - [Preventing Retries](#preventing-retries)
+       - [Executing Scripted Runs](#executing-scripted-runs)
+   - [Docker](#docker)
+     - [Elasticsearch Deployed In Docker Containers](#elasticsearch-deployed-in-docker-containers)
+     - [Running From A Docker Container](#running-from-a-docker-container)
+   - [File Sanitization Utility](#file-sanitization-utility)
+     - [Overview - How Sanitization Works](#overview---how-sanitization-works)
+     - [Running The Santizer](#running-the-santizer)
+       - [Sanitization Options](#sanitization-options)
+       - [Sanitization Examples](#sanitization-examples)
+   - [Extracting Time Series Diagnostics From Monitoring](#extracting-time-series-diagnostics-from-monitoring)
+     - [Monitoring Extract Overview](#monitoring-extract-overview)
+     - [Running The Extract](#running-the-extract)
+     - [Monitoring Extract Options](#monitoring-extract-options)
+     - [Monitoring Extract Examples](#monitoring-extract-examples)
+   - [Importing Extracted Monitoring Data](#importing-extracted-monitoring-data)
+     - [Monitoring Import Overview](#monitoring-import-overview)
+     - [Running The Monitoring Data Import](#running-the-monitoring-data-import)
+     - [Monitoring Data Import Options](#monitoring-data-import-options)
+     - [Monitoring Import Examples](#monitoring-import-examples)
+   - [Standard Diagnostic Troubleshooting](#standard-diagnostic-troubleshooting)
+ 
+ # Support Diagnostics Utility
+ 
+ [Click here for the latest version of the Support Diagnostics Utility](https://github.com/elastic/support-diagnostics/releases/latest)
+ 
+ The support diagnostic utility is a Java application that can interrogate a running Elasticsearch cluster, Kibana or Logstash process to obtain data about the state of the cluster at that point in time. It is compatible with all versions of Elasticsearch (including alpha, beta and release candidates), and for Logstash versions greater than v5, and for Kibana greater than v6.5.  The release version of the diagnostic is independent of the Elasticsearch, Kibana or Logstash version it is being run against. If it cannot match the targeted version it will attempt to run calls from the latest configured release. Linux, OSX, or Windows platforms are all supported, and it can be run as a standalone utility or from within a Docker container.
```

Comment:
I have not added this text for logstash, but checking the `logstash-rest.yml` : 
```
logstash_version:
  versions:
    "> 0.0.0": "/?pretty"

logstash_node:
  versions:
    "> 0.0.0": "/_node?pretty"

logstash_node_stats:
  versions:
    "> 0.0.0": "/_node/stats?pretty"

logstash_nodes_hot_threads:
  versions:
    "> 0.0.0": "/_node/hot_threads?human=true&threads=10000"

logstash_plugins:
  versions:
    "> 0.0.0": "/_node/plugins?pretty"
```
So I guess that any Logstash version will be executed... but on the documentation of Logstash I can only see the API since v5.5.

Anyway I will take the README in v8.3, and add my changes!